src/runtime.py

```diff
@@ -12,7 +14,7 @@ def run(self) -> None:
         agent = Agent()
         agent_locals = {}
         agent_builtins = {
-            'move': agent.move,
+            'move': lambda dir : Move(agent, dir).execute(),
```

Comment:
Name `dir` shadows builtin `dir` name and we've used `direction` name before. Let's stay consistent and use it here also.